packages/client/components/PokerEstimateHeaderCardParabol.tsx

```diff
@@ -90,10 +91,12 @@ interface Props {
   stage: PokerEstimateHeaderCardParabol_stage
 }
 
+type Story = Required<NonNullable<NonNullable<PokerEstimateHeaderCardParabol_stage>['story']>>
```

Comment:
Removing the innermost nonullable creates a TypeScript error on line 99: `content`, `id` and `teamId` `does not exist on type ...`. This is strange as `story` could be `null` but nothing else could be so one `NonNullable` should suffice. 

Any ideas why this might be? Alternatively, on line 99, I could do `(story as unknown) as Story` but I think the double `NonNullable` would be cleaner.